src/MiniCover/Instrumentation/CustomAssemblyResolver.cs

```diff
@@ -25,22 +46,37 @@ AssemblyDefinition GetAssembly(string file, ReaderParameters parameters)
 
         protected override AssemblyDefinition SearchDirectory(AssemblyNameReference name, IEnumerable<string> directories, ReaderParameters parameters)
         {
-            foreach (var directory in directories)
+            if (_dependencyContext != null)
             {
-                var files = FileUtils.GetFiles(null, null, $"**/{name.Name}.dll", directory);
```

Comment:
We can use Directory.GetFiles instead of using glob patterns. It should be faster and doesn't require additional libraries for glob processing.